File: ExcelDivider/src/main/java/ru/tsystems/divider/dao/api/EmployeeDao.java
package ru.tsystems.divider.dao.api;

import ru.tsystems.divider.entity.Employee;

public interface EmployeeDao extends GeneralDao<Employee> {
    /**
     * Finding employee who has specified first and second names
     *
     * @param firstName
     *            first name
     * @param secondName
     *            second name
     * @return {@link Employee}
     */
    Employee getByNames(String firstName, String secondName);
}

File: ExcelDivider/src/main/kotlin/ru/tsystems/divider/dao/api/EmployeeDao.kt
package ru.tsystems.divider.dao.api

import ru.tsystems.divider.entity.Employee

interface EmployeeDao : GeneralDao<Employee> {
    /**
     * Finding employee who has specified first and second names
     *
     * @param firstName
     * first name
     * @param secondName
     * second name
     * @return [Employee]
     */
    fun getByNamesIgnoreCase(firstName: String, secondName: String): Employee?
}
